test/unit/timer_sandbox.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const sinon = require('sinon');
+
```

Comment:
I'm okay with this approach but could another approach be only linting for global setTimeout/Immediate/Intervals in the `src` directory?  As long as Compass isn't running our tests I think that should be okay.

I don't have a strong preference, but the benefit of only linting the `src` directory is one fewer custom test configurations to maintain.  @addaleax

Reply:
@baileympearson I’d also be okay with applying the linting only there, I guess what would mean a new `.eslintrc` file inside `src`? That should be easy enough.

That being said, this particular file would still be necessary, because it’s about how sinon mocks the timers functions (which are then picked up by files in the `src/` directory)

Reply:
Oh, right.  That makes sense.

If we need this file either way, I'd rather lint everything for consistency 😃